tests/e2e/README.md

```diff
@@ -0,0 +1,87 @@
+# Running e2e Tests
+
+The section outlines how to set the env variable for running e2e test.
+
+## Building e2eTest.conf
+
+```text
```

Comment:
using `sh` here instead of `text` will render a lot nicer...

I wish there was an `ini` lang declaration, but I haven't seen one...